pacsman/utils_test.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from .utils import _scale_pixel_array_to_png, _pad_png_pixel_array
+
+
+def test_scale_pixel_array_to_png():
+    arr = np.array([[0.5, 0.5], [1., 1.]], dtype=float)
+    scaled = _scale_pixel_array_to_png(arr)
+    assert np.array_equal(scaled, np.array([[0, 0], [255, 255]]))
+
+
+def test_pad_png_pixel_array_1():
```

Comment:
Very nice; I like how you split up the functions to test everything.

Perhaps you can move these comments into the test names, so that they are visible when viewing pass/failure in purest runs?  E.g.:

`test_pad_png_pixel_array_1` -> `test_pad_png_pixel_array_already_square`
`test_pad_png_pixel_array_2` -> `test_pad_png_pixel_array_pad_right`
`test_pad_png_pixel_array_3` -> `test_pad_png_pixel_array_pad_down`